Makefile

```diff
@@ -234,6 +244,7 @@ test-only: all
 		$(CI_NATIVE_SUITES) \
 		known_issues
 
+# Used by make coverage-test
```

Comment:
maybe use backticks for \`make coverage-test\`. I read this wrong on first parse and thought it was just awkward wording